Controller/Process/Result.php

```diff
@@ -270,17 +269,17 @@ protected function _validateUpdateOrder($order, $response)
                 $this->_adyenLogger->addAdyenResult('Do nothing wait for the notification');
                 break;
             case Notification::CANCELLED:
-                $this->_adyenLogger->addAdyenResult('Cancel or Hold the order');
+                $this->_adyenLogger->addAdyenResult('Cancel or Hold the order on OFFER_CLOSED notification');
                 $result = false;
                 break;
             case Notification::REFUSED:
                 // if refused there will be a AUTHORIZATION : FALSE notification send only exception is idea
-                $this->_adyenLogger->addAdyenResult('Cancel or Hold the order');
+                $this->_adyenLogger->addAdyenResult('Cancel or Hold the order on OFFER_CLOSED notification');
```

Comment:
The AUTHORISATION notification success = false is going to cancel it

Reply:
Fixing it
```suggestion
                $this->_adyenLogger->addAdyenResult('Cancel or Hold the order on AUTHORISATION success=false notification');
```